CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- v0.1.1 Smaller fixes in quantization for floating-point scalar multiplication.
```

Comment:
No need to add version number here (e.g., it is not known...). I've removed it in #429.

Reply:
Makes sense.